src/main/controllers/BulkCreateMediaAccountsConfirmationController.ts

```diff
@@ -0,0 +1,79 @@
+import {PipRequest} from '../models/request/PipRequest';
+import {Response} from 'express';
+import {cloneDeep} from 'lodash';
+import {FileHandlingService} from '../service/fileHandlingService';
+import {CreateAccountService} from '../service/createAccountService';
+
+const createAccountService = new CreateAccountService();
+const fileHandlingService = new FileHandlingService();
+
+const bulkCreateAccountsUrl = 'bulk-create-media-accounts';
+const bulkCreateAccountsConfirmationUrl = 'bulk-create-media-accounts-confirmation';
+const bulkCreateAccountsConfirmedUrl = 'bulk-create-media-accounts-confirmed';
+
+export default class BulkCreateMediaAccountsConfirmationController {
+  public async get(req: PipRequest, res: Response): Promise<void> {
+    const formData = (req.cookies?.formCookie) ? JSON.parse(req.cookies['formCookie']) : {};
+    const fileName = formData['uploadFileName'];
+
+    const accountsToCreate =
+      fileName === undefined ? []
```

Comment:
This scenario should never happen in normal operation, so I wonder if this should go to an error page if it's undefined?